core/io/resource_uid.cpp

```diff
@@ -45,23 +45,44 @@ String ResourceUID::get_cache_file() {
 	return ProjectSettings::get_singleton()->get_project_data_path().path_join("uid_cache.bin");
 }
 
+static constexpr uint8_t uuid_characters[] = { 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', '0', '1', '2', '3', '4', '5', '6', '7', '8' };
+static constexpr uint32_t uuid_characters_nb = (sizeof(uuid_characters) / sizeof(*(uuid_characters)));
```

Comment:
I'm not clear on what `nb` means.

Reply:
that is short for number, but i'll be more explicit